### PR TITLE
add ReadOnlyRootfs test case

### PR DESCRIPTION
**Pull request by Helen-Xie**
ref #20  add ReadOnlyRootfs test case
@feiskyer @Random-Liu @xlgao-zju 

Signed-off-by: Helen-Xie <yulinxie@zju.edu.cn>

### CODE REVIEW

**Comment by feiskyer** [comment]
@Helen-Xie PR needs rebase.

**Comment by Helen-Xie** [comment]
update. Ready for review. @feiskyer

File: pkg/validate/security_context.go [hunk 1]
```diff
@@ -137,6 +141,42 @@ var _ = framework.KubeDescribe("Security Context", func() {
...
+
+			defer os.RemoveAll(hostPath) //clean up the TempDir
+
+			By("create container with ReadOnlyRootfs_true")
+			readOnlyRootfs := true
+			logPath, containerID := createReadOnlyRootfsContainer(rc, ic, podID, podConfig, "container-with-ReadOnlyRootfs-test-", readOnlyRootfs)
+
+			By("start container")
+			startContainer(rc, containerID)
+			Eventually(verifyContainerStatus(rc, containerID, runtimeapi.ContainerState_CONTAINER_RUNNING), pollTIMEOUT).Should(BeTrue())
```
Comment (feiskyer) [inline pkg/validate/security_context.go]: Container should be terminated with error.

**Comment by feiskyer** [comment]
ping @Helen-Xie

**Comment by Helen-Xie** [comment]
sorry, I have the exams in the last few days and I will update it as soon as possible. @feiskyer

**Comment by Helen-Xie** [comment]
Update. Ready for review. @feiskyer

File: pkg/validate/security_context.go [hunk 2]
```diff
@@ -137,6 +141,46 @@ var _ = framework.KubeDescribe("Security Context", func() {
...
+			defer os.RemoveAll(hostPath) //clean up the TempDir
+
+			By("create container with ReadOnlyRootfs_false")
+			readOnlyRootfs := false
+			logPath, containerID := createReadOnlyRootfsContainer(rc, ic, podID, podConfig, "container-with-ReadOnlyRootfs-false-test-", readOnlyRootfs)
+
+			By("start container")
+			startContainer(rc, containerID)
+			Eventually(func() bool {
+				return verifyContainerStatus(rc, containerID, runtimeapi.ContainerState_CONTAINER_RUNNING)
```
Comment (feiskyer) [inline pkg/validate/security_context.go]: use getContainerStatus() instead

Comment (Helen-Xie) [inline pkg/validate/security_context.go]: update.

**Comment by feiskyer** [comment]
@Helen-Xie Thanks. LGTM.

**Pull request merged**